lib/cli/command.ts

```diff
@@ -14,7 +14,7 @@ export default (argv: Arguments, callback: (XUClient, Arguments) => Promise<any>
 
   if (promise) {
     promise.then((result) => {
-      console.log(JSON.stringify(result));
+      console.log(JSON.stringify(result, null, 2));
```

Comment:
That was just for me to make the output of `xucli` more readable by adding an indent. And I left it because reading beautified JSON is much more fun.